jetstream/workflows/rerun.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: jetstream-
+spec:
+  entrypoint: jetstream
+  onExit: export-statistics   # export statistics after all analyses 
+  arguments:
+    parameters:
+    - name: experiment_slug   # set dynamically when workflow gets deployed
+  templates:
+  - name: jetstream
+    parallelism: 5  # run up to 5 containers in parallel at the same time
```

Comment:
We might need to tweak some of these parameters. I did a couple of test runs of previous days and things looked good with the workflow configs here.